<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NCRDDetailDialogUI</class>
 <widget class="QDialog" name="NCRDDetailDialogUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Detail settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="20,1">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="gbLeft">
      <property name="title">
       <string>Details</string>
      </property>
      <layout class="QHBoxLayout" name="layDetails">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="margin">
        <number>6</number>
       </property>
       <item>
        <widget class="QListView" name="listView"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="layAddBtn">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="NCRDPanelButtons" name="panelButtons" native="true"/>
         </item>
         <item>
          <spacer name="spacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>16</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Detail properties</string>
          </property>
          <layout class="QGridLayout" name="Layout_5554">
           <property name="verticalSpacing">
            <number>4</number>
           </property>
           <item row="2" column="1" colspan="2">
            <widget class="QComboBox" name="comboDataSource"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Height</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDoubleSpinBox" name="spHeight">
             <property name="suffix">
              <string> mm</string>
             </property>
             <property name="maximum">
              <double>999.990000000000009</double>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_4">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Data source</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QLineEdit" name="leDetailID"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Detail ID</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gbBreak">
          <property name="title">
           <string>Page break</string>
          </property>
          <layout class="QVBoxLayout" name="layPbreak">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Break condition (logical expression)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="pageBreakCond"/>
           </item>
           <item>
            <widget class="QCheckBox" name="cbNewPage">
             <property name="text">
              <string>Starts on new page</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Special</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Secondary data source</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Secondary data source</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="comboDataSource2"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Alternating rows</string>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <item row="0" column="1">
            <widget class="QComboBox" name="comboAlternateMode">
             <item>
              <property name="text">
               <string>Disabled</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Odd rows</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Even rows</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Alternating mode</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="alternateColor"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Background color</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QCheckBox" name="cbAlternateDefaultColor">
             <property name="text">
              <string>Use default background color</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>229</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="btnLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnGrouping">
       <property name="text">
        <string>Groups...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NCRDPanelButtons</class>
   <extends>QWidget</extends>
   <header>ncrdpanelbuttons.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>listView</tabstop>
  <tabstop>leDetailID</tabstop>
  <tabstop>spHeight</tabstop>
  <tabstop>comboDataSource</tabstop>
  <tabstop>pageBreakCond</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>btnGrouping</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NCRDDetailDialogUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NCRDDetailDialogUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
